### ISSUE TITLE
Bitping Dashboard shows Upgrade Required

### ISSUE THREAD

**Issue by engageub** [BitpingApp/Bitping-Node#11]
Hi Support Team,
Bitping Dashboard shows upgrade is required.
We using docker latest version which was updated 9 months ago.
![image](https://github.com/BitpingApp/Bitping-Node/assets/103608400/5308e40b-f4b2-491b-a99b-0ff36da8d176)

https://hub.docker.com/r/bitping/bitping-node/tags

Could you please look into this and let us know what needs to be done to resolve this issue. 

Thank you

**Comment by Firaenix** [comment]
You can click on the orange alert icon to be sent to a page where you can find update/installation instructions on how to upgrade your node from bitping-go-node to bitpingd/bitping desktop

**Comment by jgabriel98** [comment]
The problem is: this IS the last version.
I have the same problem. i've Just download the CLI for linux armv7 and it tells i need to upgrade...

**Comment by Firaenix** [comment]
Can you try hard refreshing the dashboard page? It could be cached.

**Comment by jgabriel98** [comment]
## Detailed exaplanation
Ok, the problem was: 

The README.md tells to install via a bash script.
The bitping docs, at the linux section, tells to `wget` the binary file and run it manually.
The dashboard page, when clicking on the "upgrade required", guides you to a page with docker instalation instructions.

I (and this issue OG) installed bitping following the official docs, that is, we installed the binary file and executed it.
And probably it's outadted or have different versioning rule.

Only after finding this github repo and folowing the steps from README.md i solved the issue.

----------------

## TL;DR:
All installation instructions differ, and the ones [from official docs](https://bitping.com/help/nodes/installing-the-node) is super outdated (at least for linux and raspberry).

### sugestion
 - Update the official docs to have the same install instructions as the readme.md
 - Update dashboard to check what type on install method the node user, so it can give correct update instructions. (like, don't tell the user to update by installing docker, when his node was installed with the CLI and is running as a service)

**Comment by Firaenix** [comment]
Ahh yes sorry about that, I'm working on the new documentation for the node now.
Apologies for it all being fragmented and confusing at the moment.

And thats a great point for the upgrade instructions on the dashboard 👍 I really appreciate your feedback

**Comment by Firaenix** [comment]
Docs have been updated to be in line with the Github readme
